### ISSUE TITLE
Separate 'matchMedia.addListener.js' into separate npm module or include when importing 'matchmedia-polyfill';

### ISSUE THREAD

**Issue by charlesdeuter** [paulirish/matchMedia.js#75]
```
require('matchmedia-polyfill');
```

doesn't work.

```
require('matchmedia-polyfill');
require('../node_modules/matchmedia-polyfill/matchMedia.addListener');
```

does work.

Don't want to gripe too much about nothing, but it would be convenient to either include the addListener polyfill in with the main file or to split it out into it's own npm module so that it can be required more conveniently. 

additionally some extra documentation about the addListener polyfill might be nice.

**Comment by jcfrancisco** [comment]
I use ```jsrequire('matchmedia-polyfill/matchMedia.addListener');`

**Comment by jcfrancisco** [comment]
I use
```js
  require('matchmedia-polyfill');
  require('matchmedia-polyfill/matchMedia.addListener');
```

But +1 on additional documentation, so that it's clear that you need to include both, for example.

**Comment by paulirish** [comment]
fixed in cd9df4c529a3c5c95c6d66b77bb7cc3d2419c460
